episodes/data-metadata.Rmd

```diff
@@ -0,0 +1,342 @@
+11. so even though lab notes allow for some kind of documentation, they are largely unstructured and subjective, rarely following templates or guidelines. The continous note taking makes the documentation hard to retrieve. They are physically seperated from the data they describe and remain in the hand of the experimentator or institution hosting the experiments, which makes them hard to share. And let's admit it, handwriting can be a pain to read.
+12. One way to store your documentation with your data, is README-style metadata... 
+13. Just like taking lab notes, Readme-style files can be subjectively written natural text documents. 
+14. You can also find templates for Readme-style metadata records. Sticking to these templates will allow for increased structure in your documentation records. Now that we document our data digitally, we have increased options to link the metadata record to the described dataset, for example by naming them accordingly...
+15. if you now want to find the results of a specific experiment on your computer, a local keyword search can help you to identify the dataset.
+16. To sum up, digital documentation of your metadata in form of Readme-style metadata records, facilitates the linkage of data and metadata, allows for local searchability and increased readability, and the digital nature of the Readme file simplifies targeted sharing of the documentation. However, these records are still rather unstructured and subjective, and only allow for keyword searches.
+17. If you want to have a look on a well-designed template for Readme-style metadata records, a recommended link is provided in the Episode description.
+18. (<i>allow some time for questions</i>)
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+**Handwritten (lab) notes** Handwritten (lab) notes are still a common
+practice in many scientific disciplines. These notes are easy to take
+during data generation. The greatest disadvantage, however, is the
+physical separation from the data itself and the difficulty to find,
+store and share these information. Often, handwritten lab notes do not
+follow a predictable structure and, hence, are hard to interpret and
+sometimes even hard to read.
+
+**Readme style text documents** Recording your metadata (additionally)
+in a digital readme style text document comes with one great advantage:
+the metadata can be associated and stored directly with the experimental
+data. Readme style metadata best practices include:[^5]
+
+[^5]: Chadwick, I. (2016). "Research Data Management: guide to
+    writing"readme" type metadata." The Open University.
+    <https://www.open.ac.uk/library-research-support/sites/www.open.ac.uk.library-research-support/files/files/RDM-Guidelines-for-creating-readme-style-metadata.pdf>
+
+<ul>
+
+<li>creating one readme file for each data file, whenever possible.</li>
```

Comment:
I imagine that it's hard to give general advice which makes sense for all use cases because of how diverse they are, though I like your suggestion of using "data set" instead of "data file".